e2e-tests/gef/cypress/integration/unissued-facilities-change-all-MIN.spec.js

```diff
@@ -154,16 +154,16 @@ context('Unissued Facilities MIN - change all to issued from unissued table', ()
       aboutFacilityUnissued.issueDateError().contains('The issue date must not be before the date of the inclusion notice submission date');
       aboutFacilityUnissued.errorSummary().contains('The issue date must not be before the date of the inclusion notice submission date');
 
-      // entering date more than 3 months from notice date
+      // entering issue date in the future
```

Comment:
Do we still need to test entering a date more than 3 months from notice date?

Reply:
since this one is a new/slightly different validation file, thought it is good to cover everything incase something breaks in the future: hence more than 3 months error testing

Reply:
My misunderstanding it should throw error for any date in the future so tomorrow makes sense 👍